plugins/grpc/client/scripts/batch_decode.py

```diff
@@ -75,24 +85,34 @@ def transcribe_audio(audio_stream, model: str, language_code: str, sample_rate=8
 
     return parse_response(response)
 
+def stream_and_transcribe(args):
```

Comment:
Can we make this function more legible? It seems unusable outside of the parallel context, maybe have more verbose arguments (with types) rather than accepting a tuple of 2 tuples.